WBooks/LIbrary/MainMenuController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  MainMenu.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 22/05/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class MainMenuController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    private lazy var _view: MainMenuView = MainMenuView.loadFromNib()!
+    
+    private let cellSpacingHeight: CGFloat = 10
+    
+    var books: [Book] = []
+    
+    init() {
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        view = _view
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        books = createArray()
+        _view.tableView.delegate = self
```

Comment:
Here you car configuring lot of tableView properties. I would be nice if you set all of this in a different function like `func configureTableView()` and call it in `viewDidLoad()`